src/lambdas/initiateAthenaQuery/createQuerySql.ts

```diff
@@ -41,7 +39,13 @@ export const createQuerySql = (
     'ATHENA_TABLE_NAME'
   )}`
 
-  const queryString = `SELECT ${sqlSelectStatement} FROM ${dataSource} WHERE ${sqlWhereStatement}`
+  const queryString = `SELECT ${sqlSelectStatement} FROM ${dataSource} WHERE ${sqlWhereStatement} AND datetime >= ? AND datetime <= ?`
+
+  const formattedDateFrom = formatDateFrom(requestData.dateFrom)
+  const formattedDateTo = formatDateTo(requestData.dateTo)
+
+  identifiers.push(formattedDateFrom)
```

Comment:
could we give this a name like parameters/athenaParams now it encompasses more things?

Is it worth generating all of them in a single function so we can look in one place and easily see what they are and the sequence they are added.

Reply:
I will make these changes - will use queryParameters and create a single function